resotocore/tests/resotocore/merge_outer_edge/merge_outer_edge_handler_test.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+import asyncio
+from pytest import fixture
+from tests.resotocore.task.task_handler_test import task_handler
+from resotocore.merge_outer_edge.merge_outer_edge_handler import MergeOuterEdgesHandler
+from resotocore.message_bus import Action, MessageBus
+from resotocore.task.task_handler import TaskHandlerService
+from resotocore.task.subscribers import SubscriptionHandler
+from tests.resotocore.message_bus_test import message_bus, all_events, wait_for_message
+from tests.resotocore.db.runningtaskdb_test import running_task_db
+
+from resotocore.task.task_description import (
+    Workflow,
+    Step,
+    PerformAction,
+    EventTrigger,
+    StepErrorBehaviour,
+    TimeTrigger,
+    Job,
+    TaskSurpassBehaviour,
+    ExecuteCommand,
+)
+
+from tests.resotocore.db.graphdb_test import (
+    filled_graph_db,
+    graph_db,
+    test_db,
+    foo_model,
+    foo_kinds,
+    system_db,
+    local_client,
+)
+
+# noinspection PyUnresolvedReferences
+from tests.resotocore.cli.cli_test import cli, cli_deps
+
+# noinspection PyUnresolvedReferences
+from tests.resotocore.analytics import event_sender
+
+# noinspection PyUnresolvedReferences
+from tests.resotocore.worker_task_queue_test import worker, task_queue, performed_by, incoming_tasks
+
+# noinspection PyUnresolvedReferences
+from tests.resotocore.query.template_expander_test import expander
+
+# noinspection PyUnresolvedReferences
+from tests.resotocore.config.config_handler_service_test import config_handler
+from tests.resotocore.db.entitydb import InMemoryDb
+
+# noinspection PyUnresolvedReferences
+from tests.resotocore.web.certificate_handler_test import cert_handler
+
+# noinspection PyUnresolvedReferences
+from tests.resotocore.task.task_handler_test import test_workflow, subscription_handler, job_db
+
+
+@fixture()
+async def merge_handler(
+    message_bus: MessageBus,
+    subscription_handler: SubscriptionHandler,
+    task_handler: TaskHandlerService,
+) -> MergeOuterEdgesHandler:
+    handler = MergeOuterEdgesHandler(message_bus, subscription_handler, task_handler)
+    return handler
```

Comment:
```suggestion
    handler.start()
    yield handler
    handler.stop()
```
make it a context manager